multiplatform/colcon_test/action.yml

```
@@ -0,0 +1,60 @@
+name: 'colcon_test'
+description: 'Import some repositories and their versions from a yaml file'
+
+inputs:
+
+  colcon_test_args:
+    description: 'args to pass to colcon test command (use ctest_args to set ctest arguments)'
+    required: false
+    default: ''
+
+  colcon_test_args_default:
+    description: 'Default args to pass to colcon test command (use ctest_args to set ctest arguments)'
+    required: false
+    default: '--event-handlers=console_direct+ --return-code-on-test-failure'
+
+  ctest_args:
+    description: 'args to pass to colcon test command'
+    required: false
+    default: ''
+
+  ctest_args_default:
+    description: 'Default args to pass to colcon test command'
+    required: false
+    default: '--timeout 60'
+
+  workspace:
+    description: 'Workspace where built is done'
```

```suggestion
    description: 'Current workspace'
```

Please, apply also to related platform actions.

---

Then I suggest 
```suggestion
    description: 'Workspace where build has been done.
```